kytos/cli/commands/controller/api.py

```diff
@@ -0,0 +1,26 @@
+"""Translate cli commands to non-cli code."""
+import subprocess
+
+
+class ControllerAPI:
+    """Handle KytosController comamnds."""
+
+    @staticmethod
+    def start():
+        """Start KytosController."""
+        subprocess.run(["kytosd"])
```

Comment:
Did you tried to run this command, when kytosd is already runing ?

Reply:
Besides that, kytos-utils is made to run in a different controller. What happen when kytosd is running in a different server ?

Reply:
This PR only works in local server. If kytosd is called when another kytosd is already running the message below will be displayed.

`The process is already running with pid 24702.`